javascript/javascript3/week1/coolApi.js

```diff
@@ -0,0 +1,19 @@
+//The Fetch API allows to asynchronously request for a resource.
+fetch("https://www.thecocktaildb.com/api/json/v1/1/random.php")//fetch method helps to return data from Api to response object.
+    .then((response) => response.json())//To get the actual data, we have to call the methods of the Response object => json()
+    .then((coctailRecipes) => {
+        //we will get the actual data (coctailRecipes) like a boject 
+        //we can access the values of object by using keys.
+        console.log(coctailRecipes);
+        const h2 = document.createElement("h2");
+        h2.innerHTML = coctailRecipes.drinks[0].strDrink;
```

Comment:
It would be best to use innerText do display the h2. 

Review the mdn documentation regarding innerHTML and the Security considerations:

https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations

Reply:
Thanks Ricardo for your time and good suggestions.